--- testsuite	2017-03-14 01:30:07.080352704 -0700
+++ testsuite.bkp	2017-03-14 01:29:03.710461315 -0700
@@ -3331,51 +3331,6 @@
     echo
   fi
 
-  {
-cat <<_ASUNAME
-## --------- ##
-## Platform. ##
-## --------- ##
-
-hostname = `(hostname || uname -n) 2>/dev/null | sed 1q`
-uname -m = `(uname -m) 2>/dev/null || echo unknown`
-uname -r = `(uname -r) 2>/dev/null || echo unknown`
-uname -s = `(uname -s) 2>/dev/null || echo unknown`
-uname -v = `(uname -v) 2>/dev/null || echo unknown`
-
-/usr/bin/uname -p = `(/usr/bin/uname -p) 2>/dev/null || echo unknown`
-/bin/uname -X     = `(/bin/uname -X) 2>/dev/null     || echo unknown`
-
-/bin/arch              = `(/bin/arch) 2>/dev/null              || echo unknown`
-/usr/bin/arch -k       = `(/usr/bin/arch -k) 2>/dev/null       || echo unknown`
-/usr/convex/getsysinfo = `(/usr/convex/getsysinfo) 2>/dev/null || echo unknown`
-/usr/bin/hostinfo      = `(/usr/bin/hostinfo) 2>/dev/null      || echo unknown`
-/bin/machine           = `(/bin/machine) 2>/dev/null           || echo unknown`
-/usr/bin/oslevel       = `(/usr/bin/oslevel) 2>/dev/null       || echo unknown`
-/bin/universe          = `(/bin/universe) 2>/dev/null          || echo unknown`
-
-_ASUNAME
-
-as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-for as_dir in $PATH
-do
-  IFS=$as_save_IFS
-  test -z "$as_dir" && as_dir=.
-    $as_echo "PATH: $as_dir"
-  done
-IFS=$as_save_IFS
-
-}
-  echo
-
-  # Contents of the config files.
-  for at_file in atconfig atlocal
-  do
-    test -r $at_file || continue
-    $as_echo "$as_me: $at_file:"
-    sed 's/^/| /' $at_file
-    echo
-  done
 } >&5
 
 
@@ -3566,16 +3521,6 @@
   fi
 done
 
-{
-  $as_echo "## ------------------ ##
-## Running the tests. ##
-## ------------------ ##"
-} >&5
-
-at_start_date=`date`
-at_start_time=`date +%s 2>/dev/null`
-$as_echo "$as_me: starting at: $at_start_date" >&5
-
 # Create the master directory if it doesn't already exist.
 as_dir="$at_suite_dir"; as_fn_mkdir_p ||
   as_fn_error $? "cannot create \`$at_suite_dir'" "$LINENO" 5
@@ -3746,31 +3691,31 @@
   $as_echo_n "$at_group. $at_setup_line: " >> "$at_group_log"
   case $at_xfail:$at_status in
     yes:0)
-	at_msg="UNEXPECTED PASS"
+	at_msg="XPASS"
 	at_res=xpass
 	at_errexit=$at_errexit_p
 	at_color=$at_red
 	;;
     no:0)
-	at_msg="ok"
+	at_msg="PASS"
 	at_res=pass
 	at_errexit=false
 	at_color=$at_grn
 	;;
     *:77)
-	at_msg='skipped ('`cat "$at_check_line_file"`')'
+	at_msg='skipped'
 	at_res=skip
 	at_errexit=false
 	at_color=$at_blu
 	;;
     no:* | *:99)
-	at_msg='FAILED ('`cat "$at_check_line_file"`')'
+	at_msg='FAILED'
 	at_res=fail
 	at_errexit=$at_errexit_p
 	at_color=$at_red
 	;;
     yes:*)
-	at_msg='expected failure ('`cat "$at_check_line_file"`')'
+	at_msg='XFAIL'
 	at_res=xfail
 	at_errexit=false
 	at_color=$at_lgn
@@ -3784,7 +3729,7 @@
     # Make sure there is a separator even with long titles.
     $as_echo " $at_color$at_msg$at_std"
   fi
-  at_log_msg="$at_group. $at_desc ($at_setup_line): $at_msg"
+  at_log_msg="$at_group. $at_desc: $at_msg"
   case $at_status in
     0|77)
       # $at_times_file is only available if the group succeeded.
@@ -3792,11 +3737,8 @@
       # is written in the global log separately.  But we also
       # write to the group log in case they're using -d.
       if test -f "$at_times_file"; then
-	at_log_msg="$at_log_msg     ("`sed 1d "$at_times_file"`')'
 	rm -f "$at_times_file"
       fi
-      $as_echo "$at_log_msg" >> "$at_group_log"
-      $as_echo "$at_log_msg" >&5
 
       # Cleanup the group directory, unless the user wants the files
       # or the success was unexpected.
@@ -3990,22 +3932,6 @@
 cd "$at_dir"
 rm -rf "$at_helper_dir"
 
-# Compute the duration of the suite.
-at_stop_date=`date`
-at_stop_time=`date +%s 2>/dev/null`
-$as_echo "$as_me: ending at: $at_stop_date" >&5
-case $at_start_time,$at_stop_time in
-  [0-9]*,[0-9]*)
-    as_fn_arith $at_stop_time - $at_start_time && at_duration_s=$as_val
-    as_fn_arith $at_duration_s / 60 && at_duration_m=$as_val
-    as_fn_arith $at_duration_m / 60 && at_duration_h=$as_val
-    as_fn_arith $at_duration_s % 60 && at_duration_s=$as_val
-    as_fn_arith $at_duration_m % 60 && at_duration_m=$as_val
-    at_duration="${at_duration_h}h ${at_duration_m}m ${at_duration_s}s"
-    $as_echo "$as_me: test suite duration: $at_duration" >&5
-    ;;
-esac
-
 echo
 $as_echo "## ------------- ##
 ## Test results. ##
@@ -4134,13 +4060,13 @@
       done
       echo
     fi
-    if test -n "$at_top_srcdir"; then
-      sed 'h;s/./-/g;s/^.../## /;s/...$/ ##/;p;x;p;x' <<_ASBOX
-## ${at_top_build_prefix}config.log ##
-_ASBOX
-      sed 's/^/| /' ${at_top_build_prefix}config.log
-      echo
-    fi
+##    if test -n "$at_top_srcdir"; then
+##      sed 'h;s/./-/g;s/^.../## /;s/...$/ ##/;p;x;p;x' <<_ASBOX
+#### ${at_top_build_prefix}config.log ##
+##_ASBOX
+##      sed 's/^/| /' ${at_top_build_prefix}config.log
+##      echo
+##    fi
   } >&5
 
   sed 'h;s/./-/g;s/^.../## /;s/...$/ ##/;p;x;p;x' <<_ASBOX
