diff --git a/gc_cpp.cc b/gc_cpp.cc
index 341805a6..fb6b0a01 100644
--- a/gc_cpp.cc
+++ b/gc_cpp.cc
@@ -38,7 +38,7 @@ built-in "new" and "delete".
 #endif
 #include "gc_cpp.h"
 
-#if !(defined(_MSC_VER) || defined(__DMC__)) || defined(GC_NO_INLINE_STD_NEW)
+#if !defined(SKIP_CPP_DEFINITIONS) && (!(defined(_MSC_VER) || defined(__DMC__)) || defined(GC_NO_INLINE_STD_NEW))
 
 #if defined(GC_NEW_ABORTS_ON_OOM) || defined(_LIBCPP_NO_EXCEPTIONS)
 # define GC_ALLOCATOR_THROW_OR_ABORT() GC_abort_on_oom()
